### PR TITLE
Henry/held piece

### PR DESCRIPTION
**Pull request by henrytran28**
Made board a class
Added a basic window that renders the held piece.

Please review Senpai

### CODE REVIEW

File: src/board.lua [hunk 1]
```diff
@@ -102,7 +119,8 @@ function Board:render()
     self:updateMatrices()
 
     -- Render the background
-    self:renderBackground()
+    self:renderBackground(properties.XOFFSET, properties.YOFFSET,
```
[the remark this answers is not in the document]
Comment (henrytran28) [reply src/board.lua]: Sure! I will change it up 👍

[the remark this answers is not in the document]
Comment (ctlai95) [reply src/board.lua]: Maybe `GameplayInfo` can have its own background for now thats separate from this `renderBackground` function

File: src/tetrion/tetrion.lua [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    return tetrion
+end
+
+function Tetrion:render()
+    -- Render the backdrop
+    love.graphics.setColor(colors.CHARCOAL)
+    love.graphics.rectangle("fill", 0, 0, 210, 600)
+
+    -- Render the held piece background
+    self:renderHeldPiecePreview()
```
Comment (ctlai95) [inline src/tetrion/tetrion.lua]:
```suggestion
    self:renderHeldPieceBackground()
```

File: src/tetrion/tetrion.lua [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+    -- Render the held piece
+    self:renderHeldPiece()
+end
+
+function Tetrion:renderHeldPiecePreview()
```
Comment (ctlai95) [inline src/tetrion/tetrion.lua]:
```suggestion
function Tetrion:renderHeldPieceBackground()
```

File: src/tetrion/tetrion.lua [hunk 4]
```diff
@@ -0,0 +1,45 @@
+local board = require("board/board")
+local utils = require("utils/utils")
+local Tetromino = require("tetromino/tetromino")
+local properties = require("tetromino/properties")
+local colors = require("colors/colors")
+local constants = require("constants")
```
Comment (ctlai95) [inline src/tetrion/tetrion.lua]:
```suggestion
```

File: src/tetrion/tetrion.lua [hunk 5]
```diff
@@ -0,0 +1,45 @@
+local board = require("board/board")
+local utils = require("utils/utils")
```
Comment (ctlai95) [inline src/tetrion/tetrion.lua]:
```suggestion
```

**Pull request merged**